daily-rotate-file.js

```diff
@@ -91,7 +91,8 @@ var DailyRotateFile = function (options) {
             extension: options.extension ? options.extension : '',
             create_symlink: options.createSymlink ? options.createSymlink : false,
             symlink_name: options.symlinkName ? options.symlinkName : 'current.log',
-            watch_log: options.watchLog ? options.watchLog : false
+            watch_log: options.watchLog ? options.watchLog : false,
+            audit_hash_type: options.auditHashType || 'sha256'
```

Comment:
I was not sure about default value to be used for audit_hash_type option. Hence used 'sha256' as it is mostly recommended and considered as strong hashing algorithm.

Reply:
Seems like a sensible default, but I'd recommend matching the default-setting format of the others, for consistency and to help avoid type confusion (about the boolean-type return value of a boolean OR operation).

Reply:
Thanks for the comment!! Updated it as current  default-setting format